src/pages/Issue/components/IssueTags.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC, Dispatch, SetStateAction } from 'react';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
```

Comment:
```suggestion
import {Paper, Grid} from '@material-ui/core';
```

구조분해할당으로 가져오면 한 줄에 표현할 수 있을 것 같습니닷!

Reply:
넵 알겠습니당